### PR TITLE
Adding FormattedDateTime component

### PR DESCRIPTION
**Pull request by sebastiankessel**
@dshahin Here it is.... 
- [ ] do this
- [x] then this

### CODE REVIEW

**Comment by dshahin** [comment]
@jholmes612 what do you think about this PR?
My thoughts are that this may be a candidate for a CSComponents repo, rather than filling making the user install too many files they may not use.  What do you think?

**Comment by jholmes612** [comment]
A components library makes sense to me. I think keeping things somewhat granular is good so that you can bring in just what you need to an org.

**Comment by sebastiankessel** [comment]
@dshahin @jholmes612  Who is creating the new repo so I can put these ones in there? You? Me?

**Comment by dshahin** [comment]
@sebastiankessel I have created a new repo:
https://github.com/CodeScience/cs-comp

please add your code there via a PR and I will merge it.

Thanks!

File: src/classes/FormattedDateTimeController.cls [hunk 1]
```diff
@@ -0,0 +1,16 @@
+public class FormattedDateTimeController {
```
Comment (ledfordbk) [inline src/classes/FormattedDateTimeController.cls]: Classes must use `with sharing` and never omit the sharing clause

Comment (ledfordbk) [inline src/classes/FormattedDateTimeController.cls]: Classes must use `with sharing`

**Comment by sebastiankessel** [comment]
5 Years old....

**Pull request closed without merging**